src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -181,8 +181,11 @@ public void UpdateDescriptionForModel(MappingDataFormatParameter mappingSource)
          }
          else
          {
-            column.Unit = _mappingParameterEditorPresenter.Unit;  
-            column.Dimension = _mappingParameterEditorPresenter.Dimension;
+            column.Unit = _mappingParameterEditorPresenter.Unit;
+            column.Dimension = _columnInfos
```

Comment:
THis set needs to be commented. too much nested info that makes it very hard to follow. 
Specifically, why is ` _mappingParameterEditorPresenter.Dimension;` not enough
also you are using, FIrst, are you sure to find one?.
Last when comparing , we use string.equals which deals with null properly (I may be wrong but I think == does not) . To verify :)